ee/clickhouse/sql/funnels/funnel.py

```diff
@@ -1,5 +1,5 @@
 FUNNEL_PERSONS_BY_STEP_SQL = """
-SELECT aggregation_target AS actor_id {extra_fields}
+SELECT aggregation_target AS actor_id{matching_events_select_statement} {extra_fields}
```

Comment:
double checking: is the no space between actor_id{matching...} safe?

Reply:
This was to decrease the amount of changes to the snapshots. Because there's a comma it should be safe not to have a space, but I just added it anyway bc not wanting to change the snapshot isn't a great reason.